docs/handbook/sunray.md

```diff
@@ -0,0 +1,359 @@
+            # new persistent configuration
+            ipadm delete-if ${INTF} 2>/dev/null
+            ipadm create-if ${INTF}
+-           ipadm create-addr -T static -a local=${IPADDR}/${MASKBITS} ${INTF}/v4static
++           ipadm create-addr -T static -a local=${IPADDR}/${MASKBITS} ${INTF}/v4utadm
+          else
+            # ifconfig the new interface
+            ifconfig ${INTF} plumb >/dev/null 2>&1 ;
+@@ -2711,7 +2711,8 @@ function DoList {
+        if [[ $BLOCKTYPE = "interface" ]] ; then
+          if Intf=${UT_DHCP_INTERFACE} ; then
+            if [[ ! -s ${HOSTNAME_R}.${Intf} ]]; then
+-               IntfAddr=""*MISSING*""
++                IntfAddr=`ipadm show-addr -p -o ADDR ${Intf}/ | awk '{split($1,a,"/"); print a[1]}'`
++               # IntfAddr=""*MISSING*""
+            else
+                typeset H=`getIfname ${Intf}`
+                IntfAddr=`getent hosts $H | awk '{ print $1 }'`
+@@ -2824,6 +2825,7 @@ function DoList {
+
+            if [[ -n "$Intf" ]]; then
+                # private interconnect, need to get the hostname and ip
++echo IntfAddr ${IntfAddr}
+                if [[ -n "${IntfAddr}" ]]; then
+                    HostName=$(getent hosts ${IntfAddr} | awk '{print $2}')
+                fi
+@@ -3156,10 +3158,10 @@ function DoPrint {
+        # Need to catch the case where hostname.<intf> exists but the hostname
+        # defined in the file may not be configured locally in the hosts file.
+        #
+-       if [ ! -f ${HOSTNAME_R}.${INTF} ]; then
+-           print -u2 "Error: missing \"${HOSTNAME_R}.${INTF}\" file."
+-           return 1
+-       fi
++       # if [ ! -f ${HOSTNAME_R}.${INTF} ]; then
++       #     print -u2 "Error: missing \"${HOSTNAME_R}.${INTF}\" file."
++       #     return 1
++       # fi
+        H=`getIfname ${INTF}`
+        if [ -z "${H}" ]; then
+            print -u2 "Error: interface ${INTF} is partially configured.  Hostname not" \
+@@ -3453,19 +3455,7 @@ function getIfname {
+        awk '{print $1 ; exit}' "${HOSTNAME_R}.$1"
+        return
+     else
+-       # With ipadm, there's no way to create a persistent
+-       # configuration without it immediately being reflected in the
+-       # runtime configuration, nor does ipadm provide any convenient
+-       # way to find the IPv4 addrobj for the interface, so just
+-       # check runtime configuration via ifconfig.
+-       IPADDR=$(ifconfig $1 | awk '
+-               /[      ]inet /{
+-                       for (i = 1; i < NF; ++i) {
+-                               if ($i == "inet") {
+-                                       print $(i+1)
+-                               }
+-                       }
+-               }')
++       IPADDR=`ipadm show-addr -p -o ADDR $1/ | awk '{split($1,a,"/"); print a[1] }'`
+        if [ -z "$IPADDR" ]; then
+            print -u2 "Error in parsing of ifconfig addresses"
+            exit 2
+```
+
+### JRE
+
+The SunRay Software needs the Sun/Oracle JRE 1.7
```

Comment:
Please, put period (`.`) here at the end of the sentence and at other places too.

Can we use our `runtime/java/openjdk7`? If not link to a usable Oracle JRE would be helpful.